### PR TITLE
Removed ET SYA App and API from Flux v1 Kustomization

### PR DESCRIPTION
**Pull request by petermoores-version1**
### JIRA link (if applicable) ###
N/A

### Change description ###
Removed ET SYA App and API from Flux v1 Kustomization because they are deploying to Demo environment when not required currently.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by petermoores-version1** [comment]
@reespozzi - Hi Rees, as requested I have created this PR to remove ET SYA App and API pods from the Demo clusters. I can only find in Flux V1 config as per this PR and not in Flux V2 config. Please can you check this looks correct and approve/comment? Thanks

**Review by gauravhmcts — APPROVED** [review]

File: k8s/namespaces/et/kustomization.yaml [hunk 1]
```diff
@@ -6,5 +6,3 @@ bases:
 # Warning : Adding a file here, adds to all environments to which you add your kustomization.
 - et-cos/et-cos.yaml
 - et-msg-handler/et-msg-handler.yaml
-- et-sya-api/et-sya-api.yaml
```
Comment (reespozzi) [inline k8s/namespaces/et/kustomization.yaml:9]: Removing from here will remove et-sya app generally, I think the other two changes are enough

Comment (petermoores-version1) [reply k8s/namespaces/et/kustomization.yaml:9]: Re-added here.

Comment (reespozzi) [reply k8s/namespaces/et/kustomization.yaml:9]: Could you also re-ad the et-sya app too please, I meant to highlight both sorry

**Review by reespozzi — APPROVED** [review]

**Review by jthmcts — APPROVED** [review]

**Pull request merged**